challenge_0/csharp/sven/README.md

```diff
@@ -0,0 +1,7 @@
+#Challenge_0 Solution
+
+To compile run "csc.exe src\HelloWorld.cs" and run the resulting file "HelloWorld.exe".
+
+csc can be found in <windows installation folder>\Microsoft.NET\<Framework bitness>\<.NET version> 
+(e.g. C:\Windows\Microsoft.NET\Framework64\v4.0.30319)
+or you can open a visual studio command prompt which adds the correct paths to its environment.
```

Comment:
For future reference, under OSX and Linux we can either install Mono or .NET directly. With Mono, use `mcs *.cs` to build and `mono *.exe` to run.

If you commit your `.csproj` file, the official .NET `dotnet` tool can run it as well.